src/api/product/model/product.model.ts

```diff
@@ -0,0 +1,32 @@
+import { Field, ObjectType, registerEnumType } from '@nestjs/graphql';
+import { Category } from 'src/api/category/model/category.model';
+import { ProductOrder } from 'src/api/product-order/model/product-order.model';
+import { VendorProduct } from 'src/api/vendor-product/model/vendor-product.model';
```

Comment:
This should be taken from 'src/api/vendor-product/model/' not from the file directly, this applies to all the cases